sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj

```diff
@@ -35,228 +35,228 @@
   <ItemGroup />
   <Import Project="$(StrideSdkTargets)" />
   <ItemGroup>
-    <ReferencePath Include="C:\Program Files\dotnet\packs\Microsoft.WindowsDesktop.App.Ref\6.0.0\ref\net6.0\Accessibility.dll" />
+    <ReferencePath Include="C:\Program Files\dotnet\packs\Microsoft.WindowsDesktop.App.Ref\8.0.0\ref\net8.0\Accessibility.dll" />
```

Comment:
That file looks like a left-over that shouldn't even exist. What is it doing here?